suse_migration_services/prechecks/sshd.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2025 SUSE LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>#
+import logging
+import re
+
+from suse_migration_services.defaults import Defaults
+from suse_migration_services.command import Command
+
+log = logging.getLogger(Defaults.get_migration_log_name())
+
+
+def root_login(migration_system=False):
+    """
+    Check whether sshd is enabled and root login is allowed. If so,
+    warn user that they might want to enable it back before starting
+    the migration.
+    """
+    system_root = Defaults.get_system_root_path() if migration_system else '/'
+    sshd_enabled = Command.run(
+        [
+            'systemctl',
+            '--root', system_root,
+            '--quiet',
+            'is-enabled', 'sshd'
+        ]
+    ).returncode == 0
```

Comment:
If you use sshd_enabled in a bool condition check, as you do, then please turn it into a bool prior use

```
some = Command.run(...)
some_has_failed = bool(some.returncode)

if some_has_failed:
    # do something when failed
```
or the respective notation for "command has failed equals something is enabled or not"

The way it is written here is imho hard to read

Reply:
Also you need to add

```
raise_on_error=False
```

to the ```Command.run``` invocation as we don't want to raise in case of failure